src/lib/y2storage/dialogs/guided_setup/select_scheme.rb

```diff
@@ -201,7 +199,7 @@ def base_help_text
               "so make sure not to lose it!</i>" \
               "</p>"
             ),
-            disk_encryption_label: WIDGET_LABELS[:use_disk_encryption]
+            disk_encryption_label: _(WIDGET_LABELS[:use_disk_encryption])
```

Comment:
Hmmm, reviewing this I just noticed `:use_disk_encryption` is not a key on that hash. Can you verify how the help text looks?

Reply:
Oh, good point, I did not check the keys. Now it looks OK.